src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs

```diff
@@ -177,11 +179,26 @@ public void Run(string[] args)
             {
                 if (windowRectangle != null)
                 {
-                    application = Application.NewApplication(args, stylesheet, windowMode, windowRectangle);
+                    if (coreTask != null)
+                    {
+                        application = Application.NewApplication(args, stylesheet, windowMode, windowRectangle, true);
+                    }
+                    else
+                    {
+                        application = Application.NewApplication(args, stylesheet, windowMode, windowRectangle);
+                    }
                 }
                 else
                 {
-                    application = Application.NewApplication(args, stylesheet, windowMode);
+                    if (coreTask != null)
+                    {
+                        // The Rectangle(0, 0, 0, 0) means that want to use the full screen size window at 0,0.
+                        application = Application.NewApplication(args, stylesheet, windowMode, new Rectangle(0, 0, 0, 0), true);
```

Comment:
Could you remove this warning message?
`Call System.IDisposable.Dispose on object created by 'new Rectangle(0, 0, 0, 0)' before all references to it are out of scope.`

For example,
```suggestion
              using (Rectangle rec = new Rectangle(0, 0, 0, 0))
              {
                        application = Application.NewApplication(args, stylesheet, windowMode, rec, true);
```